### ISSUE TITLE
Release throws 404

### ISSUE THREAD

**Issue by FerhatPflegia** [LogRocket/logrocket-cli#14]
Everytime I follow the JS Source Maps documentation and I do 
`logrocket release b620fm --apikey="your:api:key"` 
with my API key and my release string I always receive

    Creating release: 0.1.0 ... 
    Could not create release: 0.1.0
    404 Not Found
    Could not complete request.

Am I doing something wrong or is there an issue with the cli?

**Comment by eranimo** [comment]
Are you behind a firewall, pihole, VPN or something similar that might be blocking traffic to LogRocket?

**Comment by FerhatPflegia** [comment]
No, we don't use a firewall, phiole or VPN. We want to run it via GitHub Action. And locally it does not work either.

**Comment by alebiavati** [comment]
I'm having the same issu
<img width="813" alt="Screen Shot 2021-10-16 at 2 23 35 PM" src="https://user-images.githubusercontent.com/832191/137602464-327928d6-414c-4e19-8060-2409da52251f.png">
e

**Comment by rishabhgupta** [comment]
I was facing a similar issue 
```
Creating release: b620fm ...
Could not create release: b620fm
404 Not Found
Could not complete request.
```
It was happening because I was  using the `AppID` instead of `API Key`.
